server/tests/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.6'
```

Comment:
I would add some text on the of the top of this file to explain what its used for..